salt/states/zabbix_host.py

```diff
@@ -33,12 +34,15 @@ def present(host, groups, interfaces, **kwargs):
     :param groups: groupids of host groups to add the host to
     :param interfaces: interfaces to be created for the host
     :param proxy_host: Optional proxy name or proxyid to monitor host
-    :param inventory: Optional list of inventory names and values
+    :param inventory: Optional list or dictionary of inventory names and values
     :param _connection_user: Optional - zabbix user (can also be set in opts or pillar, see module's docstring)
     :param _connection_password: Optional - zabbix password (can also be set in opts or pillar, see module's docstring)
     :param _connection_url: Optional - url of zabbix frontend (can also be set in opts, pillar, see module's docstring)
     :param visible_name: Optional - string with visible name of the host, use 'visible_name' instead of 'name' \
     parameter to not mess with value supplied from Salt sls file.
+    :param inventory_clean: Optional - Boolean value that selects if the current inventory will be cleaned and \
+    overwritten by the declared inventory list (True); or if the inventory will be kept and only updated with \
```

Comment:
**question** are the backslashes necessary?

Reply:
(I see they're elsewhere... I'm just pretty sure that sphinx will remove them :shrug: )

Reply:
I don't know if they are needed, I only followed the `:param visible_name` that was already there.

Reply:
Removed the backslashes.